### PR TITLE
ready_revision data model

### PR DESCRIPTION
**Pull request by lemald**
#### Summary of changes
**Asana Ticket:** [🏹 Data model updates](https://app.asana.com/0/584764604969369/1191956107867590/f)

Per earlier discussion, I'm going to include the migration in the same PR since it should be very fast.

#### Reviewer Checklist
- [ ] Meets ticket's acceptance criteria
- [ ] Any new or changed functions have typespecs
- [ ] Tests were added for any new functionality (don't just rely on Codecov)
- [ ] This branch was deployed to the staging environment and is currently running with no unexpected increase in warnings, and no errors or crashes.

### CODE REVIEW

File: assets/src/disruptions/disruptionIndex.tsx [hunk 1]
```diff
@@ -288,8 +288,8 @@ const DisruptionIndex = () => {
   React.useEffect(() => {
     apiGet<JsonApiResponse>({
       url:
-        "/api/disruptions?only_published=" +
```
Comment (losvedir) [inline assets/src/disruptions/disruptionIndex.tsx:291]: Ah, will this require a gtfs_creator change in tandem then?

Comment (lemald) [reply assets/src/disruptions/disruptionIndex.tsx:291]: Aha, yes, thank you for the reminder... that occurred to me earlier but I completely forgot. I'll open a PR on that side in a bit.

**Comment by lemald** [comment]
> Let's come up with a plan to get this merged in tandem with the `gtfs_creator` side.

If I'm not mistaken we want to coordinate the prod Arrow deploy with the gtfs_creator merge, correct? It looks like all of the gtfs_creator environments are currently pointed to Arrow prod, or at least I don't see anything currently to point gtfs_creator dev-green to arrow-dev.

**Comment by losvedir** [comment]
Ah, right. It should be free to merge whenever then.

In terms of then deploying to prod, we just have to coordinate with merging your `gtfs_creator` branch and updating the environment variable in semaphore at the same time, right? I have meetings for much of the remainder of the afternoon, but I think it's okay to do that tomorrow morning.

**Comment by lemald** [comment]
> Ah, right. It should be free to merge whenever then.
> 
> In terms of then deploying to prod, we just have to coordinate with merging your `gtfs_creator` branch and updating the environment variable in semaphore at the same time, right? I have meetings for much of the remainder of the afternoon, but I think it's okay to do that tomorrow morning.

Yup, agreed both that those are the steps we'll need to do and that it's fine to wait until tomorrow morning.

**Pull request merged**